db/migrate/20151114125209_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,19 @@
+class CreateFriendlyIdSlugs < ActiveRecord::Migration
+  def change
+    create_table :friendly_id_slugs do |t|
+      t.string   :slug,           :null => false
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Put one space between the method name and the first argument.